src/components/PeopleFilters.tsx

```diff
@@ -1,89 +1,150 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import { SetStateAction, useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+
+type Props = {
+  urlSlug: string | undefined;
+  searchParams: URLSearchParams;
+  setSearchParams: (value: string) => void;
+};
+
+const centuriesList: string[] = ['16', '17', '18', '19', '20'];
```

Comment:
not great idea to hardcode centuries - what if people db will be supplied with people from other centuries?

construct the centuries array - i'd say it's better to construct it a level where you have the people array and then pass it towards this component.